include/bsp/xi_bsp_fwu.h

```diff
@@ -84,6 +84,23 @@ void xi_bsp_fwu_on_package_download_failure();
 
 /**
  * @function
+ * @brief Used by the Xively C Client to determine which resource to download
+ * next. This function is invoked ONLY when XI_SFT_BSP_CHOOSE_DOWNLOAD_ORDER
+ * is defined as a preprocessor definition when compiling the Xively C
+ * Client library.
+ *
```

Comment:
I thought that this function probably wouldn't be used on most platforms, so having it disabled by default could save some footprint, runtime memory usage, and complexity.  Thoughts?